src/work-allocation-2/resolvers/task.resolver.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { RouterStateSnapshot } from '@angular/router';
```

Comment:
_nitpick:_
Should be 2 space indenting.

Reply:
Done